### PR TITLE
[CDS-68459]: Git clone file content output changes

### PR DESCRIPTION
**Pull request by sainath-b**

### CODE REVIEW

**Review by rohitk-h — DISMISSED** [review]

File: posix/clone [hunk 1]
```diff
@@ -103,3 +103,10 @@ tag)
 	clone-commit
 	;;
 esac
+
+for path in ${DRONE_OUTPUT_FILE_PATHS_CONTENT//,/ }
+do
+    # call your procedure/other scripts here below
+    FILE_CONTENT=`cat $path`
+    echo "$path=$FILE_CONTENT" >> $DRONE_OUTPUT
```
Comment (shubham149) [inline posix/clone:111]: Multi-line outputs don't work well currently. What's the usecase here?

Comment (sainath-b) [reply posix/clone]: we want to resolve expressions in manager, so we want to send some files to manager using plugin outputs.

Comment (sainath-b) [reply posix/clone]: @shubham149 Changed the approach to use base64 as discussed. I didn't encode key. Encoded only content.

File: posix/clone [hunk 2]
```diff
@@ -103,3 +103,15 @@ tag)
 	clone-commit
 	;;
 esac
+
+if [[ ! -z "$DRONE_OUTPUT_FILE_PATHS_CONTENT" ]]
```
Comment (shubham149) [inline posix/clone:107]: Use PLUGIN_OUTPUT_FILE_PATHS_CONTENT instead of DRONE_OUTPUT_FILE_PATHS_CONTENT

File: posix/clone [hunk 3]
```diff
@@ -103,3 +103,15 @@ tag)
...
+then
+  for FILE_PATH in ${DRONE_OUTPUT_FILE_PATHS_CONTENT//,/ }
+  do
+      # call your procedure/other scripts here below
+      FILE_CONTENT="$(<"$FILE_PATH")"
+      FILE_CONTENT_BASE_64=`echo $FILE_CONTENT | base64`
+      printf "writing drone outputs with file path content %s=%s" $FILE_PATH $FILE_CONTENT_BASE_64
```
Comment (shubham149) [inline posix/clone:114]: printf works best with bash but this script runs with bin/sh https://stackoverflow.com/a/25894347

Comment (shubham149) [reply posix/clone:114]: can we remove the printf statement

Comment (sainath-b) [reply posix/clone:114]: I will address this comment @shubham149

File: posix/clone [hunk 4]
```diff
@@ -103,3 +103,15 @@ tag)
...
 esac
+
+if [[ ! -z "$PLUGIN_OUTPUT_FILE_PATHS_CONTENT" ]]
+then
+  for FILE_PATH in ${PLUGIN_OUTPUT_FILE_PATHS_CONTENT//,/ }
+  do
+      # call your procedure/other scripts here below
+      FILE_CONTENT="$(<"$FILE_PATH")"
+      FILE_CONTENT_BASE_64=`echo $FILE_CONTENT | base64`
+      echo "writing drone outputs with file path content $FILE_PATH=$FILE_CONTENT_BASE_64"
```
Comment (shubham149) [inline posix/clone:114]: do we need this line?

**Review by shubham149 — APPROVED** [review]

**Review by shubham149 — APPROVED** [review]

**Pull request merged**